x-pack/test/security_solution_cypress/cypress/tasks/host_risk.ts

```diff
@@ -15,9 +15,22 @@ import {
   RISK_DETAILS_NAV,
 } from '../screens/hosts/host_risk';
 
-export const navigateToHostRiskDetailTab = () => {
+export const navigateToHostRiskDetailTab = (attempts = 3) => {
```

Comment:
not my proudest work but I have no idea why sometimes the tab doesn't click, and I think these tests are useful so here we are

Reply:
humm, I thought this file was about the old host details, and that we should remove it?